supply_chain_settings/__openerp__.py

```diff
@@ -0,0 +1,48 @@
+# -*- encoding: utf-8 -*-
+###########################################################################
+#    Module Writen to OpenERP, Open Source Management Solution
+#    Copyright (C) Vauxoo (<http://vauxoo.com>).
+#    All Rights Reserved
+# ##############Credits######################################################
+#    Coded by: Hugo Francisco Adan Oliva(hugo@vauxoo.com)
+#############################################################################
+#   This program is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU Affero General Public License as published by
+#   the Free Software Foundation, either version 3 of the License, or
+#   (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU Affero General Public License for more details.
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+###############################################################################
+{
+    "name": "supplier_invoice_number_unique",
+    "version": "1.0",
+    "author": "Vauxoo",
+    "category": "Accouting",
+    "description": """
+This module adds a default purchase requisition on the wizards of
+purchase and warehouses, to generate procurements using
+calls for bids.
+    """,
+    "website": "http://www.vauxoo.com/",
+    "license": "AGPL-3",
+    "depends": [
+        "purchase_requisition"
+    ],
+    "demo": [],
+    "data": [
+        "view/res_config_view.xml"
+    ],
+    "test": [],
+    "js": [],
+    "css": [],
+    "qweb": [],
+    "installable": True,
```

Comment:
@hugho-ad please remove the auto_install and active keys. They are not used in v8